### ISSUE TITLE
FR: Per-environment configuration

### ISSUE THREAD

**Issue by daltonrooney** [codemonauts/craft-asset-autoversioning#2]
This plugin is so simple to implement and works perfectly, thanks so much! I did have one problem, which is that it's interfering with my live reload configuration with webpack, which was working before I began re-writing the URLs. I was wondering if there was any way to add a config option to disable the output on a per-environment basis so we're only re-writing URLs on staging or production? 

Right now to work around I'm just adding a whole bunch of inline conditionals but it's getting pretty messy.

I'll take a look and see if this is something I can submit a PR for, if you would consider the feature.

**Comment by daltonrooney** [comment]
I updated the PR to be a bit more flexible. Instead of enabling for production only, you can pass an array of environments where asset URLs should be modified. By default it's an empty array and the plugin will behave as normal. If you create a file in your Craft config folder called craft3-assets-autoversioning.php and add the following:

```
<?php return [
  'enableForEnv' => ['staging','production']
];
```

The output will only be modified in staging and production environments, but not in dev.

**Comment by kringkaste** [comment]
We just released version 2.0.0 for Craft 4 which supports environment variables to activate/deactivate the versioning.
